README.md

```diff
@@ -84,7 +84,9 @@ with Miri.
 
 ## Using Rustup To Specify a Specific Nightly
 
-To target a specific nightly, modify the above instructions as follows.
+To target a specific nightly, modify the above instructions as follows. It is recommended
+to use the nightly specified in the `rust-version` file in this repo, since that is the
+most recent nightly supported by Miri.
```

Comment:
Maybe I should bold this? It's pretty far down. Or maybe I should move it to the top of the README?

Reply:
I would get rid of this entire section, and instead write "Building Miri" and the following sections in a way that explicitly specifies `nightly` in every step (we have to, with `rust-toolchain` gone!). Then I think we can avoid repeating these instructions here?

Reply:
Sounds good.